packages/jmix-front-generator/src/common/studio/studio-model.ts

```diff
@@ -3,37 +3,111 @@ import {RestService, RestServiceMethod} from "../model/cuba-model";
 
 export const enum StudioTemplatePropertyType {
   ENTITY = 'ENTITY',
-  VIEW = 'VIEW',
-  NESTED_ENTITY_VIEW = 'NESTED_ENTITY_VIEW',
   STRING = 'STRING',
   BOOLEAN = 'BOOLEAN',
   INTEGER = 'INTEGER',
   OPTION = 'OPTION',
-  MULTI_OPTION = 'MULTI_OPTION',
-  REST_QUERY = 'REST_QUERY',
-  REST_SERVICE_METHOD = 'REST_SERVICE_METHOD',
   POLYMER_COMPONENT_NAME = 'POLYMER_COMPONENT_NAME',
   PASSWORD = 'PASSWORD',
   GRAPHQL_QUERY = 'GRAPHQL_QUERY',
   MENU_ITEM = 'MENU_ITEM',
   ATTRIBUTE = 'ATTRIBUTE',
   ATTRIBUTES_ARRAY = 'ATTRIBUTES_ARRAY',
+
+  // deprecated types
+  MULTI_OPTION = 'MULTI_OPTION',
```

Comment:
Why is multi-option deprecated?

Reply:
In our templates, we don't use it. I can keep it as common studio property.

Reply:
I think it shouldn't be deprecated.